OpenRA.Test/OpenRA.Test.csproj

```diff
@@ -14,26 +14,26 @@
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|x86'">
     <DebugSymbols>true</DebugSymbols>
-    <OutputPath>..\</OutputPath>
+    <OutputPath>.</OutputPath>
```

Comment:
This broke the Windows build. https://ci.appveyor.com/project/OpenRA/openra/build/1.0.1028

Please don't commit any of these unrelated changes to the project files.